### ISSUE TITLE
Slight alignment issues with swal2-success-circular-line-right when using toast.

### ISSUE THREAD

**Issue by 1ftskank** [sweetalert2/sweetalert2#878]
I've noticed that when showing a success toast message, after the animation of the green tick has finished, there is still a tiny sliver of green visible in the bottom right corner of the circle surrounding the tick:

![image](https://user-images.githubusercontent.com/22635845/35738335-783266f4-0826-11e8-8d40-dfb3f7681918.png)

It looks like the css for `top` and `left` are off by a pixel:

```
.swal2-popup.swal2-toast .swal2-success [class^='swal2-success-circular-line'][class$='right'] {
        border-radius: 0 64px 64px 0;
        top: -5px;
        left: 14px;
        -webkit-transform-origin: 0 32px;
                transform-origin: 0 32px; }
```

`top` should be `-4px` and `left` should be `15px` to match the the similar css styles for:
`.swal2-popup.swal2-toast .swal2-success [class^='swal2-success-circular-line'][class$='left']`

**Comment by limonte** [comment]
Thanks for reporting @1ftskank! Should be fixed in https://github.com/sweetalert2/sweetalert2/pull/879

Toasts is still a new feature, we didn't promote it yet. Any bug-reports about toasts are more than welcome!

**Comment by 1ftskank** [comment]
No worries, it's a nice feature!

**Comment by limonte** [comment]
Thanks, I was against it because of code complexity, but @acupajoe decided to prove me wrong and now we have both modals and toasts in 1 package :muscle:

**Comment by limonte** [comment]
Fixed in v7.9.1 :tada:
